packages/create-playkit-plugin/createPlaykitPlugin.js

```diff
@@ -129,19 +133,74 @@ if (program.info) {
         .then(console.log);
 }
 
-if (typeof projectName === 'undefined') {
-    console.error('Please specify the project directory:');
-    console.log(
-        `  ${chalk.cyan(program.name())} ${chalk.green('<project-directory>')}`
-    );
-    console.log();
-    console.log('For example:');
-    console.log(`  ${chalk.cyan(program.name())} ${chalk.green('my-react-app')}`);
-    console.log();
-    console.log(
-        `Run ${chalk.cyan(`${program.name()} --help`)} to see all options.`
-    );
-    process.exit(1);
+async function askProjectOptions() {
+    const validation = (pattern = /^[a-zA-Z0-9-]*$/g) => input => {
+        if (!input) {
+            return "This field should not be empty.";
+        }
+
+        if (!pattern.test(input)) {
+            return "It is not valid! Please try again.";
+        }
+
+        return true;
+    };
+
+    if (projectName) {
+        projectDestination = path.resolve(projectName);
+        projectName = path.basename(projectDestination);
+    }
+
+    const askName = {
+        type: "text",
+        name: "name",
+        message: `Please specify the project name:`,
+        validate: validation(),
+        initial: projectName,
+    };
+
+    const askNpmName = name => ({
+        type: "text",
+        name: "npmName",
+        message: "Please specify the plugin NPM name:",
+        validate: validation(/^(@[a-z-]+\/[a-z-]+|[a-z-]+)*$/g),
```

Comment:
I'm not sure the asterisk is relevant here

![image](https://user-images.githubusercontent.com/4751797/70038694-58265500-15c1-11ea-90da-202e0ff42531.png)